src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartLegendConfiguration.tsx

```diff
@@ -354,6 +354,7 @@ const ChartLegendConfiguration = ({
                         <div className="dfe-flex dfe-justify-content--space-between">
                           <div className={styles.labelInput}>
                             <FormFieldTextInput
+                              addOnContainerClassName="dfe-flex-grow--1"
```

Comment:
Feels like it'd be sensible to add `dfe-flex-grow--1` as one of the conditional classes in `FormBaseInput`:

```tsx
<div
  className={classNames(
    {
      'dfe-flex': !!addOn,
      'dfe-flex-grow--1': !addOn,
    },
    addOnContainerClassName,
  )}
>
```

However, I'm also wondering if we could go further with this and just remove the container div if `addOn` is present or not e.g.

```tsx
// FormBaseInput.tsx

const input = (
  <input
    // rest of props
  />
);

return (
  <>
    // ... some stuff
    {addOn ? (
      <div className={classNames('dfe-flex', addOnContainerClassName)}>
        {input}
        {addOn}
      </div>
    ) : (
      input
    )}
  </>
);

```